src/cultionet/data/create.py

```diff
@@ -331,7 +331,7 @@ def create_image_vars(
                            .clip(0, 1))
 
             # Get the band count per index
-            nbands = int(src_ts.gw.nbands / len(list(set([Path(fn).parent.name for fn in image]))))
+            nbands = len(list(set([Path(fn).parent.name for fn in image])))
```

Comment:
Thanks for this fix to allow the use of single-band inputs.

The variable name is correct with this change. However, this change impacts the downstream ['flip' augmentation](https://github.com/jgrss/cultionet/blob/10b8826fc1847e494c81a8dc979da290a667401e/src/cultionet/augment/augmentation.py#L144) because I was previously using `nbands` as `ntime`.

See additional modifications in [PR #19 ](https://github.com/jgrss/cultionet/pull/19)

Reply:
So I actually wanted to ask you about flipfb, I currently have it turned off. Is the intention of it to reverse time, or is it to reverse the bands? If the latter, I imagine it would be best to have it off if I am using NRGB, correct? (since I don't want it reversing those?) Thanks!